.github/workflows/trivy-nightly-scan.yaml

```diff
@@ -0,0 +1,34 @@
+name: Trivy Nightly Scan
+on:
+  schedule:
+    - cron: '0 2 * * *' # run at 2 AM UTC
+
```

Comment:
The last comment, please add 
```
permissions: read-all
```

Refer to https://github.com/etcd-io/etcd/blob/ebe653da2ceff783e040bc06fe6fa6ffdde8441b/.github/workflows/govuln.yaml#L3

Reply:
We require a write permission to upload the results to the security tab, Will that conflict with this permission?

Reply:
`permissions: read-all` means the workflow can't change the repository. It can upload results.